route-rs-runtime/src/pipeline/mod.rs

```diff
@@ -1,2 +1,8 @@
+//! # What are they?
+//!
+//! Pipelines are abstractions used by graphgen to define a grouping of links that are run by the runtime, in our case Tokio. The graphgen
+//! program takes the user provided graph, and generates a pipeline. It connects input and output channels to the pipe, so that it can be
+//! hooked up to packet sources and sinks, and drops the pipeline into the runtime. Generally can be abstracted to the concept, router.
```

Comment:
> Generally can be abstracted to the concept, router.

Can you clarify this sentence? As in, the full Router program is composed of Pipelines?